db/migrate/20210126181906_remove_aal_from_service_providers.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveAalFromServiceProviders < ActiveRecord::Migration[6.1]
+  def change
+    safety_assured { remove_column :service_providers, :aal, :integer }
```

Comment:
Won't this have the same problem on production we had on `int`? Until the old instances are scaled out, they will be trying to access the `aal` field which is no longer there...

Reply:
Yep, #4642 was brought into this branch to address that